internal/service/federatedsettingsidentityprovider/resource_federated_settings_identity_provider_test.go

```diff
@@ -11,13 +11,13 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
-	matlas "go.mongodb.org/atlas/mongodbatlas"
+	"go.mongodb.org/atlas-sdk/v20231115002/admin"
 )
 
 func TestAccFedRSFederatedSettingsIdentityProvider_basic(t *testing.T) {
```

Comment:
When do you plan to add migration tests? If planning to merge this to master right away we should add some as part of this PR